tools/lkl/lib/hijack/hijack.c

```diff
@@ -283,3 +283,20 @@ int epoll_ctl(int epollfd, int op, int fd, struct epoll_event *event)
 
 	return lkl_call(__lkl__NR_epoll_ctl, 4, epollfd, op, fd, event);
 }
+
+int eventfd(unsigned int count, int flags)
+{
+	return lkl_sys_eventfd2(count, flags);
```

Comment:
I was going to make a PR for this, but I think that either the `__attribute__((constructor))`s or the `CHECK_HOST_CALL`s should be removed --- preferably the former because afaik constructor functions are run once per thread rather than once per process (so spawning a new worker thread in hijacked code will trigger a wave of `dlsym`s).